src/test/java/dev/openfeature/sdk/OpenFeatureClientTest.java

```diff
@@ -27,13 +35,16 @@ void shouldNotThrowExceptionIfHookHasDifferentTypeArgumentThanHookContext() {
         FlagEvaluationDetails<Boolean> actual = client.getBooleanDetails("feature key", Boolean.FALSE);
 
         assertThat(actual.getValue()).isTrue();
-        assertThat(TEST_LOGGER.getLoggingEvents()).filteredOn(event -> event.getLevel().equals(Level.ERROR)).isEmpty();
+        // I dislike this, but given the mocking tools available, there's no way that I know of to say "no errors were logged"
+        Mockito.verify(logger, never()).error(any());
+        Mockito.verify(logger, never()).error(anyString(), any(Throwable.class));
+        Mockito.verify(logger, never()).error(anyString(), any(Object.class));
+        Mockito.verify(logger, never()).error(anyString(), any(), any());
+        Mockito.verify(logger, never()).error(anyString(), any(), any());
```

Comment:
I agree it's not ideal. It's probably not a thing people assert in tests very often.